pixeltable/env.py

```diff
@@ -930,12 +941,19 @@ def __init_spacy(self) -> None:
         dependency, we install it programmatically here. This should cause no problems, since the model packages
         have no sub-dependencies (in fact, this is how spaCy normally manages its model resources).
         """
+        import contextlib
+        import io
+
         import spacy
         from spacy.cli.download import download
 
         spacy_model = 'en_core_web_sm'
         self._logger.info(f'Ensuring spaCy model is installed: {spacy_model}')
-        download(spacy_model)
+
+        # prevent download() from hanging due to its progress bar, which conflicts with our use of Rich Progress
```

Comment:
We should really just get rid of the spacy auto-downloader (I could take that on as a future rough-edges ticket)